nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -34,6 +34,9 @@ Keep an old app running and swap the ports when the new one is booted.
           host = "*4";
           port = 6981;
         }];
+        # rotate-logs = false; emits keter logs and it's applications to stderr
+        # which allows journald to capture them.
+        rotate-logs = false;
       };
```

Comment:
I'm not sure if `default` for this option is going to be recursive. Please observe [this pattern](https://github.com/ckiee/nixpkgs/blob/3e272a4f63a46813bf95e3fca23684dc8049d9a3/nixos/modules/services/web-servers/garage.nix#L33-L59) where we can document individual options of a `settings`-esque configuration.

Reply:
To be clearer, what I mean is I don't know what if `rotate-logs`'s default would stay as `false` when you change something else under `globalKeterConfig`. Like if you set:
```nix
services.keter.globalKeterConfig.some-other-option = "blep";
```
Will that keep `rotate-logs`? I'm rusty.

Reply:
I copied the pattern as you suggested.
This is much nicer in usage.